### PR TITLE
Problem: bank doesn't make use of binding contract

### PR DESCRIPTION
**Pull request by mmsqe**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: x/cronos/keeper/precompiles/bank.go [hunk 1]
```diff
@@ -122,21 +75,17 @@ func (bc *BankContract) checkBlockedAddr(addr sdk.AccAddress) error {
...
+	method, err := bankABI.MethodById(methodID)
+	if err != nil {
+		return nil, err
+	}
 	stateDB := evm.StateDB.(ExtStateDB)
 	precompileAddr := bc.Address()
-
-	switch string(methodID) {
-	case string(MintMethod.ID), string(BurnMethod.ID):
+	switch string(method.Name) {
```
Comment (mmsqe) [inline x/cronos/keeper/precompiles/bank.go:84]:
```suggestion
	switch method.Name {
```

File: scripts/run-integration-tests [hunk 2]
```diff
@@ -7,6 +7,7 @@ export TMPDIR=/tmp
 
 echo "build test contracts"
 cd ../integration_tests/contracts
+cp -r ../../x/cronos/events/bindings/src ./contracts
```
Comment (yihuang) [inline scripts/run-integration-tests:10]: Can we put a symbolic link?

Comment (mmsqe) [inline scripts/run-integration-tests:10]:
```suggestion
ln -s $(realpath "../../x/cronos/events/bindings/src") ./contracts/src
```

File: scripts/run-integration-tests [hunk 3]
```diff
@@ -7,6 +7,7 @@ export TMPDIR=/tmp
 
 echo "build test contracts"
 cd ../integration_tests/contracts
+ln -s $(realpath "../../x/cronos/events/bindings/src") ./contracts/src
```
Comment (yihuang) [inline scripts/run-integration-tests:10]: I mean commit the symbolic link file into repo, so we don't need to add this line at all ;D

Comment (mmsqe) [reply scripts/run-integration-tests:10]: why not, everyone need gen their own symlink

Comment (yihuang) [reply scripts/run-integration-tests:10]: Can we just create the file, do we have to create on the fly?

Comment (mmsqe) [reply scripts/run-integration-tests:10]: You is right, change later

**Review by yihuang — APPROVED** [review]

**Pull request merged**